src/averageAges.js

```diff
@@ -20,6 +20,21 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const sexMale = 'm';
+  const calculateYearCentury = 100;
```

Comment:
```suggestion
  const YEARS_IN_CENTURY = 100;
```
It's a constant naming convention.